app/views/widgets/select_search.scala.html

```diff
@@ -1,4 +1,4 @@
-@(field : play.api.data.Field, options: List[(String, Boolean)])(labelContent: Html)(implicit messages: Messages)
+@(field : play.api.data.Field, options: Seq[String])(labelContent: Html)(implicit messages: Messages)
```

Comment:
I was told that we want to keep widget with the same name across different project to be the same,
so probably if you need different functionality you can create different widget.